pkgs/applications/science/math/almonds/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pythonPackages, fetchFromGitHub, ncurses }:
+
+pythonPackages.buildPythonPackage {
+  name = "almonds-2015-12-27";
+  src = fetchFromGitHub {
+    owner = "Tenchi2xh";
+    repo = "Almonds";
+    rev = "e69303fddb56a5f49f99d5601e2c020692965717";
+    sha256 = "1r9amh4navp1v68v1bhpwm5vxny6jidabb9v1nnqal0wi79z6w04";
+  };
+
+  buildInputs = [
+    pythonPackages.pytest
+    ncurses
+  ];
+
+  propagatedBuildInputs = [
+    pythonPackages.pillow
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Terminal Mandelbrot fractal viewer";
+    homepage = https://github.com/Tenchi2xh/Almonds;
+    maintainers = with maintainers; [ infinisil ];
+    platforms = platforms.linux;
```

Comment:
Any reason for this?
It should build on Darwin as well. Since `buildPythonApplication` automatically sets the platform, you can just leave out `platforms`.